.github/workflows/ci.yml

```diff
@@ -57,3 +59,27 @@ jobs:
         env:
           CI_NAME: github-actions
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+      - name: Pack
+        run: dotnet pack src -c Release --no-build --output nuget
+      - name: 'Upload Artifact'
+        uses: actions/upload-artifact@v2
+        with:
+          name: nuget
+          path: nuget/
+
+  publish:
+    if: (github.event_name == 'workflow_dispatch') && (github.ref == 'refs/heads/master') && (endsWith(github.actor, '-stripe')
+    needs: [build]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download all workflow run artifacts
+      uses: actions/download-artifact@v2
+      with:
+        name: nuget
+        path: nuget
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 6.0.x
+    - name: Publish NuGet packages to NuGet
+      run: dotnet nuget push nuget/*.nupkg --api-key ${{ secrets.NUGET_KEY }} --source "nuget.org"
```

Comment:
Do we need to filter explicitly for `src\Stripe.net\bin\Release\*.nupkg` as we were doing in appveyor.yml? Or is that no longer necessary here?

Reply:
Because files are coming from the artifact they land in the nuget directory we don't need extra filtering.